### ISSUE TITLE
C5 and TDigest dependency warnings

### ISSUE THREAD

**Issue by ctaggart** [wavefrontHQ/wavefront-sdk-csharp#20]
It would be awesome to help bring those dependencies up to netstandard2.0 too.

```
warning NU1701: Package 'C5 2.3.0.1' was restored using '.NETFramework,Version=v4.6.1' instead of the project target framework '.NETCoreApp,Version=v2.1'. This package may not be fully compatible with your project.
warning NU1701: Package 'TDigest 1.0.6' was restored using '.NETFramework,Version=v4.6.1' instead of the project target framework '.NETCoreApp,Version=v2.1'. This package may not be fully compatible with your project.
```

And then the list of dependencies displayed is Visual Studio is massive. It may be due to the above.
![image](https://user-images.githubusercontent.com/80104/53121530-691a4400-3522-11e9-9ad7-a918269a7961.png)

**Comment by hanwavefront** [comment]
Thanks for the feedback.  We will reach out and work with the owners of those packages on that.

**Comment by hanwavefront** [comment]
Opened https://github.com/quantumtunneling/T-Digest.NET/issues/7 to address this.

**Comment by ByronComparisonTech** [comment]
Is there any movement on this? 
Is it possible to move away from the T-Digest dependency if they're not updating their package?

**Comment by thepeterstone** [comment]
We have released [v1.8.0-beta](https://www.nuget.org/packages/Wavefront.SDK.CSharp/1.8.0-beta) which removes the dependency on T-Digest and bumps C5 to v3.0.0-beta.
